pom.xml

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.codeforcommunity</groupId>
+  <artifactId>jumpstart-backend</artifactId>
+  <packaging>pom</packaging>
+  <version>1.0-SNAPSHOT</version>
+
+  <modules>
+    <module>api</module>
+    <module>service</module>
+  </modules>
+
+  <properties>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
+  <build>
+    <defaultGoal>clean spotless:apply install</defaultGoal>
```

Comment:
okay... what runs this? I never knew there was a default (add to comment)

Reply:
Yeah I went through and described a lot of the stuff going on in the pom. I was going to leave a lot of it for the other ticket, but I don't see a reason why it can't be done now.

Reply:
I threw that in a while ago but forgot to tell people lmao.